docs/src/api/class-clock.md

```diff
@@ -249,7 +249,7 @@ Time to be set.
 ## async method: Clock.setSystemTime
 * since: v1.45
 
-Sets current system time but does not trigger any timers.
+Sets current system time without pausing, but does not trigger any timers. You most likely don't need this.
```

Comment:
- I don't think this is helpful. We better explain a usecase here, e.g. "testing how the web page reacts to a time shift, for example switching from summer to winter time, or changing a time zone". Also, it works when paused as well.
- Can we update `setFixedTime` docs to say you should either use it, or `install()`.
- Links to the clock guide would probably be helpful here as well.

Reply:
Thanks for all the feedback. I've pushed a second attempt, how do you like that?